opentelekomcloud/resource_opentelekomcloud_fw_rule_v2.go

```diff
@@ -246,6 +251,30 @@ func resourceFWRuleV2Delete(d *schema.ResourceData, meta interface{}) error {
 	return rules.Delete(networkingClient, d.Id()).Err
 }
 
+func assignedPolicyID(nwClient *golangsdk.ServiceClient, ruleID string) (string, error) {
```

Comment:
Why we need to check this? Do you mean the policy update method not take effect for the last rule?

Reply:
The problem, rule can be removed before policy is updated (and it always does), so we should be sure to remove rule from policy.
I haven't change the logic of `resourceFWRuleV2Delete`, if was intended to work the same way before. Previously there was a check for `rule.PolicyID` to be not empty, but it is not set anywhere and not retrieved using `rules.Get`, so search for policies is required. This solution seems to be ugly from my point of view, but I can't find any other way to do that.

Reply:
So, it should be a BUG on server side right? rule.PolicyID should be set there but in fact not.

Reply:
No, it is not an issue ccording VPC spec, there is no `policy_id` in response: https://docs.otc.t-systems.com/api/vpc/vpc_firewall_0002.html